### ISSUE TITLE
Neodash Error

### ISSUE THREAD

**Issue by AnneGriff** [maurice1408/Neodash#9]
Running my big query, below. I get the error 
<img width="1479" alt="image" src="https://github.com/user-attachments/assets/869b23bf-15c2-4250-b9bb-89d9d7586537">
Sometimes the query times out, but others it runs really quickly. It runs fine on the console.  I am so nearly there. 

Hope this is what you wanted re backtick?

```

MATCH (p1:Person), (p1f:Person), (p2:Person), (p2f:Person), (p3:Person), (p3f:Person), (p4:Person), (p4f:Person), (p5:Person), (p5f:Person), 
(p6:Person), (p6f:Person), (p7:Person), (p7f:Person), (p8:Person), (p8f:Person), (p9:Person), (p9f:Person), (p10:Person), (p10f:Person), 
(p11:Person), (p11f:Person), (p11fd:Person), (p12:Person), (p12f:Person), (p13:Person), (p13f:Person),(p14:Person), (p14f:Person),(p15:Person), (p15f:Person), 
(p16:Person), (p16f:Person), (p17:Person), (p17f:Person), (p18:Person), (p18f:Person), (p19:Person), (p19f:Person),(p20:Person), (p20f:Person),
//(p21:Person), 
(p21f:Person), (p22:Person), (p22f:Person), (p23:Person), (p23f:Person), (p24:Person), (p24f:Person), 
(e31:Residence),
(o11f:Object), (u20:Publication), (u23f1:Publication), (u161:Publication), (u21f1:Publication),
(pt3a:Person), (pt3b:Person), 
(p13a:Person), (s1:Society), (s2:Society)
WHERE 
p1.name   = "George III, King of Great Britain and Ireland"									AND
p1f.name  = "Charlotte, Queen of Great Britain and Ireland and Electress of Hanover"		AND
p2.name   = "Mead, Richard"																	AND
p2f.name  = "Brydges, Cassandra, 1st Duchess of Chandos"									AND
p3.name   = "Dashwood, Francis, 11th Baron Le Despencer"									AND
p3f.name  = "Delany, Mary"																	AND
//********* to do
p4.name  =  "Fitzroy, Augustus Henry, 3rd Duke of Grafton" 									AND
p4f.name  = "Banks, Sarah Sophia" 															AND
p5.name   = "Banks, Sir Joseph"																AND
p5f.name  = "Lee, Ann" 																		AND
p6.name   = "Beckford, William Thomas"														AND
p6f.name  = "Bentinck, Margaret Cavendish, 2nd Duchess of Portland" 						AND
p7.name   = "Stuart, John, 3rd Earl of Bute"												AND
p7f.name  = "Lee, Ann" 																		AND
p8.name   = "Hamilton, Sir William"															AND
p8f.name  = "Montagu, Elizabeth" 															AND
p9.name   = "Clifford, George"																AND
p9f.name  = "Cavendish, Georgiana, 5th Duchess of Devonshire" 								AND
p10.name   = "Clifford, George"																AND
p10f.name  = "Cavendish-Bentinck, Dorothy, 3rd Duchess of Portland" 						AND
p11.name  =  "Compton, Henry, Bishop of London" 											AND
p11f.name = "Monson, Lady Anne" 															AND
p12.name =  "Solander, Daniel" 																AND
p12f.name = "Graham, Catherine" 															AND												
p13.name =  "Hughes, Rev. Griffith" 														AND
p13f.name = "Villiers, Frances, 4th Countess of Jersey" 									AND
//********* to do
p14.name =  "Worsley, Lady Seymour Dorothy" 												AND
//********* to do
p14f.name = "Furber, Robert" 																AND
//********* to do
p15.name =  "Withering, William" 															AND
p15f.name = "Hamilton, Lady Emma" 															AND
p16.name =  "Smellie, William" 																AND
p16f.name = "Beauclerk, Lady Diana" 														AND
p17.name = "Petiver, James" 																AND
//********* to do
p17f.name = "Damer, Anne Seymour" 															AND
p18.name = "Dillenius, Johann" 																AND
p18f.name = "Beauclerk, Harriot, 9th Duchess of St Albans" 									AND
p19.name =  "Graham, Dr James" 																AND
//********* to do
p19f.name = "Fox, Elizabeth Vassall, 3rd Baroness Holland" 									AND
//********* to do
p20.name = "Sherard, James" 																AND
//********* to do
p20f.name = "Jacson, Maria Elizabetha" 														AND
//********* to fix
//p21.name =  "Rousseau, Jean-Jacques" 														AND
p21f.name = "Blackwell, Elizabeth"															AND
//********* to do
p22.name =  "Darwin, Erasmus" 																AND
//********* to do
p22f.name = "Smith-Stanley, Lady Elizabeth, 12th Countess of Derby II"						AND
p23.name =  "Ehret, Georg" 																	AND
p23f.name = "Nihell, Elizabeth"																AND
p24.name =  "Hogarth, William" 																AND
p24f.name = "Battiscombe, Honor"															AND
o11f.name = "Monson Bengal Insect Collection" 												AND
e31.name = "West Wycombe Park" 																AND
u20.name = "Hortus Elthamensis" 															AND
u23f1.name = "The Art of Midwifery" 														AND
u161.name = "The Theory and Practice of Midwifery" 											AND
u21f1.name = "A Curious Herbal"																AND
p11fd.name = "Vane, William Harry, 1st Duke of Cleveland" 									AND	
pt3a.name = "Cozens, Alexander"																AND
pt3b.name = "Chambers, Sir William" 														AND
p13a.name = "Bray, Rev. Thomas" 															AND
s1.name = "Temple Coffee House Botanic Club"												AND
s2.name = "Society for the Propagation of the Gospel in Foreign Parts"	

//
// 
//
OPTIONAL MATCH 
//
// GEORGE III TO BANKS
//
r1a = shortestPath((p1:Person)-[*..3]-(p5:Person)),
//
// GEORGE III TO QUEEN CHARLOTTE
//
r1b = shortestPath((p1:Person)-[*..3]-(p1f:Person)),
//
// GEORGE III TO BATTISCOMBE
//
r1c = shortestPath((p1:Person)-[*..3]-(p24f:Person)),
//
// GEORGE III TO VILLIERS
//
r1d = shortestPath((p1:Person)-[*..3]-(p13f:Person)),
//
// GEORGE III TO BECKFORD
//
r1e = (pt3a:Person)-[:Taught]-(r3:Person),
r1f = (pt3b:Person)-[:Taught]-(r3:Person),
r1g = (pt3a:Person)-[:Taught]-(r6:Person),
r1h = (pt3b:Person)-[:Taught]-(r6:Person),
//
// QUEEN CHARLOTTE TO DELANY
//
r1fa = shortestPath((p1f:Person)-[*..3]-(p3f:Person)),
//
// QUEEN CHARLOTTE TO DIANA BEAUCLERK
//
r1fb = shortestPath((p1f:Person)-[*..3]-(p16f:Person)),
//
// DASHWOOD TO DELANY *** WEST WYCOMBE
//
r3a = (p3:Person)-[:Owned]-(e31:Residence),
r3fa = (p3f:Person)-[:Visited]->(e31:Residence),
//
// DASHWOOD TO BUTE
//
r3b = shortestPath((p3:Person)-[*..3]-(p7:Person)),
//
// DELANY TO DoPORTLAND
//
r3fb = shortestPath((p3f:Person)-[*..3]-(p6f:Person)),
//
// DoPORTLAND TO DoPORTLAND
//
r3fc = shortestPath((p3f:Person)-[*..3]-(p10f:Person)),
//
// ******************************DoPORTLAND TO ROUSSEAU
//
//r3fd = shortestPath((p3f:Person)-[*..3]-(p21:Person)),
// ******************************
//
// HAMILTON TO HAMILTON
//
r8a = shortestPath((p8:Person)-[*..3]-(p15f:Person)),
//
// HAMILTON TO DoDEVONSHIRE
//
r8b = shortestPath((p8:Person)-[*..3]-(p9f:Person)),
//
// SOLANDER TO BANKS
// 
r12a = shortestPath((p12:Person)-[*..3]-(p5:Person)),
// 
// SOLANDER TO S-S BANKS
//
r12b = shortestPath((p12:Person)-[*..3]-(p4f:Person)),
// 
// SOLANDER TO DIANA BEAUCLERK
//
r12c = shortestPath((p12:Person)-[*..3]-(p16f:Person)),
//
// BANKS TO S-S BANKS
//
r5a = shortestPath((p5:Person)-[*..3]-(p4f:Person)),
//
// MONSON TO SOLANDER
//
r11fa = shortestPath((p12:Person)-[*..3]-(p11f:Person)),
//
// MONSON TO LEE INSECT COLLECTION
// 
r11fb = (p11f:Person)-[:Created]->(o11f:Object),
r11fc = (o11f)-[:Bequeathed_to]->(p5f:Person),
//
// MONSON TO HARRIOT BEAUCLERK (COUTTS) BY LOVER)
//
r11fd = (p11f:Person)-[:Kin]->(p11fd:Person),
r11fe = (p11ff:Person)-[:Spouse]->(p11fd:Person),
r11ff = (p11ff:Person)-[:Lover]->(p11fg:Person),
r11fg = (p18f:Person)-[:Spouse]->(p11fg:Person),
//
// HUGHES TO COMPTON BY SPG
//
r13a = (p13:Person)-[:Worked_for]->(s2:Society),
r13b = (p13a:Person)-[:Founded]->(s2:Society),
r13c = (p11:Person)-[:Patron]->(p13a:Person),
//
// PETIVER TO COMPTON TEMPLE HOUSE COFFEE
//
//r17a = shortestPath((p17:Person)-[*..5]-(p11:Person)),
r17a = (p17:Person)-[:Member]->(s1:Society),
r11a = (p11:Person)-[:Member]->(s1:Society),
//
//
// GRAHAM TO GRAHAM (MACAULAY)
//
r19a = shortestPath((p19:Person)-[*..3]-(p12f:Person)),
//
// GRAHAM TO DoDEVONSHIRE
//
r19b = shortestPath((p19:Person)-[*..3]-(p9f:Person)),
//
// GRAHAM TO HAMILTON
//
r19c = shortestPath((p19:Person)-[*..3]-(p15f:Person)),
//
// SHERARD TO DILLENIUS HORTUS ELTHAMENSIS
//
r20a = (p20:Person)-[:Commissioned]->(u20:Publication),
r20b = (p18:Person)-[:Wrote]->(u20),
//
// SHERARD TO PETIVER KIN
//
r20c = shortestPath((p20:Person)-[*..5]-(p17:Person)),
//
//
// SHERARD TO MEAD CURIOUS HERBAL
//
r20d = shortestPath((p20:Person)-[*..3]-(p2:Person)),
//
// MEAD TO DASHWOOD FOUNDLING HOSPITAL
//
r2a = shortestPath((p2:Person)-[*..3]-(p3:Person)),
r2b = shortestPath((p2:Person)-[*..3]-(p24:Person)),
//
// MEAD TO BLACKWELL
//
r2c = shortestPath((p2:Person)-[*..3]-(p21f:Person)),
r21fa = (p21f:Person)-[:Wrote]->(u21f1:Publication),
//
// BLACKWELL TO BRYDGES
//
r21fb = shortestPath((p21f:Person)-[*..3]-(p2f:Person)),
//
// BLACKWELL TO SMELLIE
//
r21fc = shortestPath((p21f:Person)-[*..3]-(p16:Person)),
//
// EHRET TO CLIFFORD
//
r23a = shortestPath((p23:Person)-[*..3]-(p9:Person)),
//
// EHRET TO DoPORTLAND
//
r23b = shortestPath((p23:Person)-[*..3]-(p6f:Person)),
//
// SMELLIE TO NIHELL ART OF MIDWIFERY RESPONDS TO 
//
r23fa = (p23f:Person)-[:Wrote]->(u23f1:Publication),
r16a = (p16:Person)-[:Wrote]->(u161:Publication),
r23fb = (u23f1:Publication)-[:Responded]->(u161:Publication)

RETURN r1a, r1b, r1c, r1d, r1e, r1f, r1g, r1h, r1fa, r1fb, r2a, r2b, r2c, r3a, r3b, r3fa, r3fb, 
r3fc, 
// *****************r3fd, 
r5a, r8a, r8b,
r11a, r11fa, r11fb, r11fc, r11fd, r11fe, r11ff, r11fg, r12a, r12b, r12c, 
r13a, r13b, r13c, 
r16a, r17a, r19a, r19b, r19c, 
r20a, r20b, 
r20c, 
r20d, 
r21fa, r21fb, r21fc, r23a, r23b, r23fa, r23fb
```

**Comment by AnneGriff** [comment]
Latest version. Had enough now! 

```
MATCH (p1:Person), (p1f:Person), (p2:Person), (p2f:Person), (p3:Person), (p3f:Person), (p4:Person), (p4f:Person), (p5:Person), (p5f:Person), 
(p6:Person), (p6f:Person), (p7:Person), (p7f:Person), (p8:Person), (p8f:Person), (p9:Person), (p9f:Person), (p10:Person), (p10f:Person), 
(p11:Person), (p11f:Person), (p11fd:Person), (p12:Person), (p12f:Person), (p13:Person), (p13f:Person),(p14:Person), (p14f:Person),(p15:Person), (p15f:Person), 
(p16:Person), (p16f:Person), (p17:Person), (p17f:Person), (p18:Person), (p18f:Person), (p19:Person), (p19f:Person),(p20:Person), (p20f:Person),
//(p21:Person), 
(p21f:Person), (p22:Person), (p22f:Person), (p23:Person), (p23f:Person), (p24:Person), (p24f:Person), 
(e31:Residence),
(o11f:Object), (u20:Publication), (u23f1:Publication), (u161:Publication), (u21f1:Publication),
(pt3a:Person), (pt3b:Person), (p19a:Person), 
(p13a:Person), (s1:Society), (s2:Society), (s3:Society), (t1:Theatre)
WHERE 
p1.name   = "George III, King of Great Britain and Ireland"									AND
p1f.name  = "Charlotte, Queen of Great Britain and Ireland and Electress of Hanover"		AND
p2.name   = "Mead, Richard"																	AND
p2f.name  = "Brydges, Cassandra, 1st Duchess of Chandos"									AND
p3.name   = "Dashwood, Francis, 11th Baron Le Despencer"									AND
p3f.name  = "Delany, Mary"																	AND
//********* to do
p4.name  =  "Fitzroy, Augustus Henry, 3rd Duke of Grafton" 									AND
p4f.name  = "Banks, Sarah Sophia" 															AND
p5.name   = "Banks, Sir Joseph"																AND
p5f.name  = "Lee, Ann" 																		AND
p6.name   = "Beckford, William Thomas"														AND
p6f.name  = "Bentinck, Margaret Cavendish, 2nd Duchess of Portland" 						AND
p7.name   = "Stuart, John, 3rd Earl of Bute"												AND
p7f.name  = "Lee, Ann" 																		AND
p8.name   = "Hamilton, Sir William"															AND
p8f.name  = "Montagu, Elizabeth" 															AND
p9.name   = "Clifford, George"																AND
p9f.name  = "Cavendish, Georgiana, 5th Duchess of Devonshire" 								AND
p10.name   = "Clifford, George"																AND
p10f.name  = "Cavendish-Bentinck, Dorothy, 3rd Duchess of Portland" 						AND
p11.name  =  "Compton, Henry, Bishop of London" 											AND
p11f.name = "Monson, Lady Anne" 															AND
p12.name =  "Solander, Daniel" 																AND
p12f.name = "Graham, Catherine" 															AND												
p13.name =  "Hughes, Rev. Griffith" 														AND
p13f.name = "Villiers, Frances, 4th Countess of Jersey" 									AND
//********* to do
p14.name =  "Worsley, Lady Seymour Dorothy" 												AND
//********* to do
p14f.name = "Furber, Robert" 																AND
p15.name =  "Withering, William" 															AND
p15f.name = "Hamilton, Lady Emma" 															AND
p16.name =  "Smellie, William" 																AND
p16f.name = "Beauclerk, Lady Diana" 														AND
p17.name  = "Petiver, James" 																AND
p17f.name = "Damer, Anne Seymour" 															AND
p18.name  = "Dillenius, Johann" 															AND
p18f.name = "Beauclerk, Harriot, 9th Duchess of St Albans" 									AND
p19.name  =  "Graham, Dr James" 															AND
//********* to do
p19f.name = "Fox, Elizabeth Vassall, 3rd Baroness Holland" 									AND
//********* to do
p20.name = "Sherard, James" 																AND
//********* to do
p20f.name = "Jacson, Maria Elizabetha" 														AND
//********* to fix
//p21.name =  "Rousseau, Jean-Jacques" 														AND
p21f.name = "Blackwell, Elizabeth"															AND
//********* to do
p22.name =  "Darwin, Erasmus" 																AND
//********* to do
p22f.name = "Smith-Stanley, Lady Elizabeth, 12th Countess of Derby II"						AND
p23.name =  "Ehret, Georg" 																	AND
p23f.name = "Nihell, Elizabeth"																AND
p24.name =  "Hogarth, William" 																AND
p24f.name = "Battiscombe, Honor"															AND
o11f.name = "Monson Bengal Insect Collection" 												AND
e31.name = "West Wycombe Park" 																AND
u20.name = "Hortus Elthamensis" 															AND
u23f1.name = "The Art of Midwifery" 														AND
u161.name = "The Theory and Practice of Midwifery" 											AND
u21f1.name = "A Curious Herbal"																AND
p11fd.name = "Vane, William Harry, 1st Duke of Cleveland" 									AND	
p19a.name  = "Maria I, Queen of Portugal" 													AND	
pt3a.name = "Cozens, Alexander"																AND
pt3b.name = "Chambers, Sir William" 														AND
p13a.name = "Bray, Rev. Thomas" 															AND
s1.name = "Temple Coffee House Botanic Club"												AND
s2.name = "Society for the Propagation of the Gospel in Foreign Parts"						AND
s3.name = "Lunar Society"																	AND
t1.name = "Theatre Royal, Drury Lane"	

//
// 
//
OPTIONAL MATCH 
//
// GEORGE III TO BANKS
//
r1a = shortestPath((p1:Person)-[*..3]-(p5:Person)),
//
// GEORGE III TO QUEEN CHARLOTTE
//
r1b = shortestPath((p1:Person)-[*..3]-(p1f:Person)),
//
// GEORGE III TO BATTISCOMBE
//
r1c = shortestPath((p1:Person)-[*..3]-(p24f:Person)),
//
// GEORGE III TO VILLIERS
//
r1d = shortestPath((p1:Person)-[*..3]-(p13f:Person)),
//
// GEORGE III TO BECKFORD
//
r1e = (pt3a:Person)-[:Taught]-(r3:Person),
r1f = (pt3b:Person)-[:Taught]-(r3:Person),
r1g = (pt3a:Person)-[:Taught]-(r6:Person),
r1h = (pt3b:Person)-[:Taught]-(r6:Person),
//
// QUEEN CHARLOTTE TO DELANY
//
r1fa = shortestPath((p1f:Person)-[*..3]-(p3f:Person)),
//
// QUEEN CHARLOTTE TO DIANA BEAUCLERK
//
r1fb = shortestPath((p1f:Person)-[*..3]-(p16f:Person)),
//
// DASHWOOD TO DELANY *** WEST WYCOMBE
//
r3a = (p3:Person)-[:Owned]-(e31:Residence),
r3fa = (p3f:Person)-[:Visited]->(e31:Residence),
//
// DASHWOOD TO BUTE
//
r3b = shortestPath((p3:Person)-[*..3]-(p7:Person)),
//
// DELANY TO DoPORTLAND
//
r3fb = shortestPath((p3f:Person)-[*..3]-(p6f:Person)),
//
// DoPORTLAND TO DoPORTLAND
//
r3fc = shortestPath((p3f:Person)-[*..3]-(p10f:Person)),
//
// ******************************DoPORTLAND TO ROUSSEAU
//
//r3fd = shortestPath((p3f:Person)-[*..3]-(p21:Person)),
// ******************************
//
// HAMILTON TO HAMILTON
//
r8a = shortestPath((p8:Person)-[*..3]-(p15f:Person)),
//
// HAMILTON TO DoDEVONSHIRE
//
r8b = shortestPath((p8:Person)-[*..3]-(p9f:Person)),
//
// DoDEVONSHIRE TO DARMER
//
r9fa = shortestPath((p9f:Person)-[*..3]-(p17f:Person)),
//
// SOLANDER TO BANKS
// 
r12a = shortestPath((p12:Person)-[*..3]-(p5:Person)),
// 
// SOLANDER TO S-S BANKS
//
r12b = shortestPath((p12:Person)-[*..3]-(p4f:Person)),
// 
// SOLANDER TO DIANA BEAUCLERK
//
r12c = shortestPath((p12:Person)-[*..3]-(p16f:Person)),
//
// BANKS TO S-S BANKS
//
r5a = shortestPath((p5:Person)-[*..3]-(p4f:Person)),
//
// MONSON TO SOLANDER
//
r11fa = shortestPath((p12:Person)-[*..3]-(p11f:Person)),
//
// MONSON TO LEE INSECT COLLECTION
// 
r11fb = (p11f:Person)-[:Created]->(o11f:Object),
r11fc = (o11f)-[:Bequeathed_to]->(p5f:Person),
//
// MONSON TO HARRIOT BEAUCLERK (COUTTS) BY LOVER)
//
r11fd = (p11f:Person)-[:Kin]->(p11fd:Person),
r11fe = (p11ff:Person)-[:Spouse]->(p11fd:Person),
r11ff = (p11ff:Person)-[:Lover]->(p11fg:Person),
r11fg = (p18f:Person)-[:Spouse]->(p11fg:Person),
//
// HUGHES TO COMPTON BY SPG
//
r13a = (p13:Person)-[:Worked_for]->(s2:Society),
r13b = (p13a:Person)-[:Founded]->(s2:Society),
r13c = (p11:Person)-[:Patron]->(p13a:Person),
//
// DARWIN TO WITHERING LUNAR SOCIETY
//
r15a = (p15:Person)-[:Member]->(s3:Society),
r15b = (p22:Person)-[:Member]->(s3:Society),
//
// PETIVER TO COMPTON TEMPLE HOUSE COFFEE
//
//r17a = shortestPath((p17:Person)-[*..5]-(p11:Person)),
r17a = (p17:Person)-[:Member]->(s1:Society),
r11a = (p11:Person)-[:Member]->(s1:Society),
//
// SMITH-STANLEY TO HARRIOT BEAUCLERK (COUTTS) BY THEATRE
//
r18fa = (p18f:Person)-[:Worked_at]->(t1:Theatre),
r18fb = (p22f:Person)-[:Worked_at]->(t1:Theatre),
//
//
// GRAHAM TO GRAHAM (MACAULAY)
//
r19a = shortestPath((p19:Person)-[*..3]-(p12f:Person)),
//
// GRAHAM TO DoDEVONSHIRE
//
r19b = shortestPath((p19:Person)-[*..3]-(p9f:Person)),
//
// GRAHAM TO HAMILTON
//
r19c = shortestPath((p19:Person)-[*..3]-(p15f:Person)),
//
// GRAHAM TO WITHERING
//
r19d = (p19:Person)-[:Worked_for]->(p19a:Person),
r19e = (p15:Person)-[:Worked_for]->(p19a:Person),
//
// SHERARD TO DILLENIUS HORTUS ELTHAMENSIS
//
r20a = (p20:Person)-[:Commissioned]->(u20:Publication),
r20b = (p18:Person)-[:Wrote]->(u20),
//
// SHERARD TO PETIVER KIN
//
r20c = shortestPath((p20:Person)-[*..5]-(p17:Person)),
//
//
// SHERARD TO MEAD CURIOUS HERBAL
//
r20d = shortestPath((p20:Person)-[*..3]-(p2:Person)),
//
// MEAD TO DASHWOOD FOUNDLING HOSPITAL
//
r2a = shortestPath((p2:Person)-[*..3]-(p3:Person)),
r2b = shortestPath((p2:Person)-[*..3]-(p24:Person)),
//
// MEAD TO BLACKWELL
//
r2c = shortestPath((p2:Person)-[*..3]-(p21f:Person)),
r21fa = (p21f:Person)-[:Wrote]->(u21f1:Publication),
//
// BLACKWELL TO BRYDGES
//
r21fb = shortestPath((p21f:Person)-[*..3]-(p2f:Person)),
//
// BLACKWELL TO SMELLIE
//
r21fc = shortestPath((p21f:Person)-[*..3]-(p16:Person)),
//
// SMITH STANLEY TO DARMER
//
r22fa = shortestPath((p22f:Person)-[*..3]-(p17f:Person)),
//
// EHRET TO CLIFFORD
//
r23a = shortestPath((p23:Person)-[*..3]-(p9:Person)),
//
// EHRET TO DoPORTLAND
//
r23b = shortestPath((p23:Person)-[*..3]-(p6f:Person)),
//
// SMELLIE TO NIHELL ART OF MIDWIFERY RESPONDS TO 
//
r23fa = (p23f:Person)-[:Wrote]->(u23f1:Publication),
r16a = (p16:Person)-[:Wrote]->(u161:Publication),
r23fb = (u23f1:Publication)-[:Responded]->(u161:Publication)

RETURN r1a, r1b, r1c, r1d, r1e, r1f, r1g, r1h, r1fa, r1fb, r2a, r2b, r2c, r3a, r3b, r3fa, r3fb, 
r3fc, 
// *****************r3fd, 
r5a, r8a, r8b, r9fa,
r11a, r11fa, r11fb, r11fc, r11fd, r11fe, r11ff, r11fg, r12a, r12b, r12c, 
r13a, r13b, r13c, 
r15a, r15b, 
r16a, r17a, 
r18fa, r18fb,
r19a, r19b, r19c, r19d, r19e,
r20a, r20b, 
r20c, 
r20d, 
r21fa, r21fb, r21fc, 
r22fa,
r23a, r23b, r23fa, r23fb
```

**Comment by maurice1408** [comment]
The above is returning nothing because this match...

`//   (p11fd:Person      {name:  "Vane, William Harry, 1st Duke of Cleveland"}),` does not find anything.

I have commented out any reference to `p11fd`, reformatted the code and saved it in this repository as `paths.cypher`, you can download it from here.

It runs in desktop and in browser against your `f5fa` instance. It's slow though, you're basically trying to do too much in 1 query. In neodash I can get it to run if ...

1. in the dashboard settings (settings wheel top right) I set the "Maximum Query Timeout" to 120 second 
2. Create a new graph report and past the `paths.cypher` code into the reoprt.

**Comment by AnneGriff** [comment]
Brilliant! That code works quite swiftly. Thank you. Is the only thing you changed making individual MATCHES for each name. I thought I was doing an efficient thing putting them all on one statement.

**Comment by AnneGriff** [comment]
I think you updated, got an email but can't see anything?

The above is returning nothing because this match... I dont understand that - I have just uncommented and the query  works fine for me. 
//   (p11fd:Person      {name:  "Vane, William Harry, 1st Duke of Cleveland"}), does not find anything.

I notice that you removed and reordered some of the returns - any reason? I have added them back and put back in my order, I can't believe that the order is important is it?
